src/sched.rs

```diff
@@ -119,3 +119,34 @@ pub fn setns(fd: RawFd, nstype: CloneFlags) -> Result<()> {
 
     Errno::result(res).map(drop)
 }
+
+#[cfg(not)]
```

Comment:
cfg what?

Reply:
I wanted to see if the tests on travis would pass if the test was disabled. It seems that with this test in place, both this test and the `sys::wait::tests::test_waitpid` test would fail in various situations, but not in a way that I could replicate locally, even when building for targets that were failing in travis.